frontend-vue/src/router.js

```diff
@@ -1,23 +1,35 @@
 import Vue from 'vue'
 import Router from 'vue-router'
 import Home from './views/Home.vue'
+import misha from '@/views/misha.vue';
 
 Vue.use(Router)
 
 export default new Router({
   routes: [
-    {
-      path: '/',
-      name: 'home',
-      component: Home
-    },
     {
       path: '/about',
       name: 'about',
       // route level code-splitting
       // this generates a separate chunk (about.[hash].js) for this route
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+    },
+    {
+      path:'/Home_page',
```

Comment:
@Reisande Can you verify that these page routes are fine?

I'm thinking that the home page should be at route `/` instead of `/Home_page` (so that users would only have to go to `thelugdndsite.com/` instead of `thelugdndsite.com/Home_page`), but I'm not sure of the routes for the character input and character output pages.

Reply:
Fixed it to just '/'